### ISSUE TITLE
Tournament statistics

### ISSUE THREAD

**Issue by Nordlandia** [cutechess/cutechess#365]
1# Total duration in days & hours and minutes

2# Average game duration in days, hours and minutes 

3# Average moves per game. In TCEC that is somewhere around 60 moves.

Can fit inside "Tournament Result" tab at the bottom.

**Comment by Ipmanchess** [comment]
4. In real time seeing Time Forfaits!!
   Now using a tool from Ferdinand who made me a bat file so that it can follow the running games and show me these Time Forfaits if any happen.

**Comment by alwey** [comment]
@Ipmanchess : Your suggestion (4) is a duplicate, please refer to #355.

**Comment by Ipmanchess** [comment]
@alwey : yes i know ..i can't pointed enough ,and that someone else is kindly to make a tool for me..but of course it would be better if it's integraded! Thanks!

**Comment by Nordlandia** [comment]
@alwey Long time, no see. 💬
